src/srs/index.ts

```diff
@@ -1,22 +1,3 @@
-export interface SRS {
-  code: string
-  name: string
-  wkt: string
-}
+export { DEFAULT_SRS, SRS } from './srs'
```

Comment:
SRS must be exported with type. can u check if there's a es lint rule that prevents exporting types without type?

e.g. https://github.com/intuit/eslint-plugin-no-explicit-type-exports

Reply:
fixed